### PR TITLE
Follow up to pull request #1366 - Centralize UAO version control

### PR DESCRIPTION
**Pull request by mattatobin**
We need both the quoted and unquoted versions of these.

Quoted for http protocol and unquoted for uaoverrides.inc

Tested and verified working on all fronts.

### CODE REVIEW

File: configure.in [hunk 1]
```diff
@@ -1887,15 +1887,19 @@ MOZILLA_SYMBOLVERSION=`$PYTHON $srcdir/python/mozbuild/mozbuild/milestone.py --t
 AC_DEFINE_UNQUOTED(MOZILLA_VERSION,"$MOZILLA_VERSION")
 AC_DEFINE_UNQUOTED(MOZILLA_VERSION_U,$MOZILLA_VERSION)
 AC_DEFINE_UNQUOTED(MOZILLA_UAVERSION,"$MOZILLA_UAVERSION")
+AC_DEFINE_UNQUOTED(MOZILLA_UAVERSION_U,$MOZILLA_UAVERSION)
 AC_SUBST(MOZILLA_SYMBOLVERSION)
 AC_SUBST(MOZILLA_UAVERSION)
```
Comment (JustOff) [inline configure.in]: AC_SUBST(MOZILLA_UAVERSION) isn't used anymore (it was added in #1366 ), so it should be deleted.

File: configure.in [hunk 2]
```diff
@@ -1887,15 +1887,19 @@ MOZILLA_SYMBOLVERSION=`$PYTHON $srcdir/python/mozbuild/mozbuild/milestone.py --t
...
+AC_SUBST(MOZILLA_UAVERSION_U)
 
 dnl ========================================================
 dnl Get Firefox compatibility version we claim in our UA
 dnl ========================================================
 MOZILLA_COMPATVERSION=`tail -n 1 "$_topsrcdir"/config/compatversion.txt 2>/dev/null || tail -1 "$_topsrcdir"/config/compatversion.txt`
 AC_DEFINE_UNQUOTED(MOZILLA_COMPATVERSION,"$MOZILLA_COMPATVERSION")
+AC_DEFINE_UNQUOTED(MOZILLA_COMPATVERSION_U,$MOZILLA_COMPATVERSION)
 AC_SUBST(MOZILLA_COMPATVERSION)
```
Comment (JustOff) [inline configure.in]: The same for AC_SUBST(MOZILLA_COMPATVERSION).

File: configure.in (hunk 1 above)
Comment (mattatobin) [reply configure.in]: I wouldn't.. Most stuff that has an AC_DEFINE has an AC_SUBST it would be wiser to leave it be.

File: configure.in (hunk 2 above)
Comment (mattatobin) [reply configure.in]: Ditto.

File: configure.in (hunk 1 above)
Comment (mattatobin) [reply configure.in]: If anything should go it should be the _U substs

Comment (JustOff) [reply configure.in]: Well, I agree, an extra define will certainly not break anything.

Comment (mattatobin) [reply configure.in]: Creating config.status errors out without accompanying AC_SUBSTs I tried all combos.. They stay.. GOOD TO MERGE!

File: configure.in (hunk 2 above)
Comment (JustOff) [reply configure.in]: Ok.

**Comment by mattatobin** [comment]
Sometimes we should leave well enough alone when it is working ;)

**Comment by JustOff** [comment]
I confirm this resolves the issue with the `uaoverrides.inc`.

**Pull request merged**